### ISSUE TITLE
Build fails on latest alpine linux (3.19.1) 

### ISSUE THREAD

**Issue by homme** [duckdb/duckdb#11254]
### What happens?

Building duckdb under alpine linux 3.19.1 fails.  This is after following the duckdb [build instructions for Alpine](https://duckdb.org/dev/building).  It appears the issue is due to undeclared symbols.

### To Reproduce

```
git clone https://github.com/duckdb/duckdb.git && \
docker run -i --rm -v $(pwd)/duckdb:/duckdb --workdir /duckdb alpine:latest <<< "apk add g++ git make cmake ninja && GEN=ninja make" 2>&1 | tee build.log
```

[build.log](https://github.com/duckdb/duckdb/files/14653134/build.log)

### OS:

Alpine Linux x86_64 Linux

### DuckDB Version:

commit e245e2927f14fb8bcbe7c794218fb73449f3dce1 (HEAD)

### DuckDB Client:

cli build

### Full Name:

Homme Zwaagstra

### Affiliation:

Strava

### Have you tried this on the latest [nightly build](https://duckdb.org/docs/installation/?version=main)?

I have tested with a nightly build

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] Yes, I have

**Comment by szarnyasg** [comment]
Hi @homme, a fix is up in #11265

**Comment by homme** [comment]
Hi @szarnyasg, many thanks for the fix: I can confirm it works :+1:.
